corehq/apps/reports/generic.py

```diff
@@ -193,7 +192,7 @@ def __str__(self):
 
     def __getstate__(self):
         """
-            For pickling the report when passing it to Celery.
+            Returns report parameters to rebuild report in Celery.
```

Comment:
__getstate__ and __setstate__ are methods specifically used for pickling, so I think now that we are moving away from that (and assuming we don't pickle reports anywhere else), it would be cool to name these methods more appropriately. I also think it is important to note whether in the method name itself or the docs for the method that it returns a json serializable dictionary.

Reply:
By the method's docs do you mean like the comment on line 195 right now, but more in depth?

Reply:
Yeah just making sure that it is noted the returned dictionary is json serializable
```
def test():
    """
    <method docs>
    """
```